README.md

```diff
@@ -232,6 +232,10 @@ e.g.
 }
 ```
 
+## Data and Telemetry
+
+The `vscode-yaml` extension collects anonymous [usage data](USAGE_DATA.md) and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.elemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-commons#how-to-disable-telemetry-reporting
```

Comment:
Is `redhat.elemetry.enabled` missing a `t`?

Reply:
That question for `vscode-commons`, as that property defined [there](https://github.com/redhat-developer/vscode-commons/blob/master/package.json#L29)

Reply:
@evidolob you made a typo `redhat.elemetry.enabled` should be `redhat.telemetry.enabled`